seed/static/seed/js/seed.js

```diff
@@ -1420,3 +1431,9 @@ SEED_app.constant('naturalSort', function (a, b) {
     else if (oFxNcL > oFyNcL) return 1;
   }
 });
+
+SEED_app.constant('seedConstants', {
```

Comment:
@axelstudios Can you chime in on this pattern?

Reply:
I'm good with this pattern (a constant that can only be accessed via dependency injection).  There are better patterns for this with TypeScript and Angular 2+, but it's way better than the global namespace pollution going on [here](https://github.com/SEED-platform/seed/blob/edd89fa9b126231e5fb50622b31c7b275ee8646d/seed/templates/seed/_js_urls.html#L5).

Technically it might be unsafe because I think you could override the original value, the way around that would be to make them functions that return a string.  You could also put these in a service.

Reply:
Refactored this pattern.
Constants are now in static/seed/js/constants.js, registered as a module, and are now injectable by constant identifier.

I opted for single value constants rather than a single injectable constant object in order to prevent accessing undefined values due to typos. I tested it and you CAN reassign values to constants, but hopefully future developers will recognize and respect the UPPERCASE_CONVENTION of constants.

I considered implementing a constants service, but decided against it b/c
1. this seems to be the prescribed way of using constants in angularjs
2. code completion is nice
3. slightly less code (e.g. MY_CONST vs seed_constants.get('MY_CONST')